InstallSpinnaker.sh

```diff
@@ -109,6 +115,43 @@ function echo_status() {
   fi
 }
 
+function process_provider_list() {
+  AWS_ENABLED=false
+  AZURE_ENABLED=false
+  GOOGLE_ENABLED=false
+
+  #check if aws enabled
```

Comment:
Could you make a loop here that processes the next token in the list?
    a=$(CLOUD_PROVIDER) should turn the space delimited list into an array
then iterate over the array and process each element individually.
    for provider in ${a[@]}; do ...; done
That way we'll be able to detect malformed strings.